invesalius/net/neuronavigation_api.py

```diff
@@ -48,3 +53,28 @@ def update_coil_pose(self, position, orientation):
                 position=position,
                 orientation=orientation,
             )
+
+    def update_coil_mesh(self, polydata):
+        wrapped = dataset_adapter.WrapDataObject(polydata)
+
+        points = np.array(wrapped.Points)
```

Comment:
I think it's better to do:

```python
points = np.asarray(wrapped.Points)
```
Using `np.asarray` it creates a numpy array using the same memory, `np.array` copies the memory to create a new array, using more memory.

Reply:
Done.